index.js

```diff
@@ -190,7 +180,7 @@ module.exports = Hermes;
 Hermes.prototype.publish = function (queueName, data) {
   /*jshint maxcomplexity:7 */
   debug('hermes publish', queueName, data);
-  if (!~this.queues.indexOf(queueName)) {
+  if (!~this.opts.queues.indexOf(queueName)) {
     throw new Error('attempting to publish to invalid queue: '+queueName);
   }
```

Comment:
in theory, you'd just have to unit test that helper function; asserting it is called for subscribe, unsubscribe, etc., would then validate that you are ensuring it's a valid queue name. really just a testing thing